app.js

```diff
@@ -0,0 +1,98 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const port = 3000;
+const bodyParser = require("body-parser");
+const School = require("./School")
+let newSchool = new School();
+const timeStamp = new Date();
+
+app.use(cors());
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.json());
+
+const allClasses = (req,res) => {
+    res.json(newSchool);
+}
+
+const classCheck = (req, res, next) => {
+    if(newSchool[classes][req.body.name]){
+        res.json({
+            error:" Info either has already been submited or missing sections",
+            timeStamp:timeStamp.toString(Date.now())
+        })
+    }else{
+        next();
+    }
+
+}
+
+const checkStudent = (req, res, next) => {
+    let enrolledStudents = newSchool[classes][`${req.params["className"]}`]["students"];
+    for(let i = 0; i < enrolledStudents.length; i++){
+        if(enrolledStudents[i]["name"] === req.body.name){
+            res.json({
+                error:"Fill out the student info",
+                timestamp: timeStamp.toString(Date.now())
+            })
+            return
+        }
+    }
+    next()
+}
+
+const addNewClass = (req, res) => {
+    newSchool.addClass(req.body.name, neq.body.teacher);
+    res.json({
+        class:{ "name": req.body.name, "teacher": req.body.teacher, "students": []},
+        message: "Added a new class",
+        timestamp: timeStamp.toString(Date.now())
+    })
+}
+
+const addNewStudent = (req, res) => {
+    newSchool.enrollStudent(req.params["className"], req.body);
```

Comment:
You should assign this to a variable and include that variable in your response as a `student`